### DISCUSSION TITLE
Living HTA?

### DISCUSSION THREAD

**Discussion by dantesalvadormd** [HTAhackathon2024/.github#20] [Ideas]
### What is the problem we're trying to solve

I've seen colleagues do a living meta-analysis. Well, what's stopping us from making living HTA? 

There will always be new clinical evidence, and by the time an HTA report is almost completed (may take a year for example), the systematic review/meta-analysis included will already have been "old" (depends on which field). Same is true for some cost elements, such as direct and indirect medical costs.

Some people already wrote about it (https://pubmed.ncbi.nlm.nih.gov/36652184/), but would it be a doable project to write skeletal codes/blocks for automating/semi-automating inputs in a living HTA? 

The outline is provided in the article, but we can perhaps select on a finished HTA (that's not written as a Living HTA) with published data/code

### How will we tackle the problem?

Write sample blocks of code for each element of living HTA

**Comment by RobertASmith** [comment]
Hi Dante,

This paper might be useful:

https://wellcomeopenresearch.org/articles/7-194

Kind regards,

Rob

**Reply by dantesalvadormd** [reply]
Awesome, thanks Robert! It just occurred to me that the "Some people..." I referred to in my initial post include you. :D
I suppose we can close this discussion now, the work (living HTA code blocks) has been done at least 2 years ago.

**Reply by RobertASmith** [reply]
I think this is still an interesting topic. The example prototype we built only ran the economic model and produced the health economic modelling report. 

It would be good to extend this to include the meta-analysis step of the pipeline (the step before the Wellcome Open Research paper prototype).

**Comment by bisacciamd** [comment]
This concept is very interesting, thanks for proposing it. I would be glad to help if you get started on streamlining the meta-analysis component at the hackathon next week.
